README.md

```diff
@@ -39,6 +38,51 @@ query = gql('''
 client.execute(query)
 ```
 
+To execute against a graphQL API. (We get the schema by using introspection).
+
+```
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+
+_transport = RequestsHTTPTransport(
```

Comment:
It would be helpful to explain that this class comes from the `requests` library, and takes the various arguments (e.g. `headers`, `auth`).